apps/console/src/features/roles/components/application-roles.tsx

```diff
@@ -112,6 +112,9 @@ export const ApplicationRoles: FunctionComponent<ApplicationRolesSettingsInterfa
     const path: string[] = history.location.pathname.split("/");
     const appId: string = path[path.length - 1].split("#")[0];
 
+    const isSubOrg: boolean = window[ "AppUtils" ].getConfig().organizationName;
+    const isReadOnly: boolean = !!isSubOrg;
```

Comment:
Shall we use the `organizationType` to check the organization is a sub org ? It would be much cleaner

Reply:
Fixed with 98166ecda2c7a50e90bc6a20b14c9e3ed008a469